manager2/src/app/user/user.component.html

```diff
@@ -4,8 +4,9 @@
           <h2><b>{{user.uid}}</b>
           <div *ngIf="subscribed && (session_user.is_admin || (config.enable_ui && config.enable_ui.newsletters))"><button class="btn" (click)="unsubscribe()"><small class="oi oi-envelope-open glyphicon glyphicon-envelope" title="The user is subscribed to the newletters" style="color:green"></small> unsubscribe</button></div>
           <div *ngIf="! subscribed"><button class="btn" (click)="subscribe()"><small class="oi oi-envelope-open glyphicon glyphicon-envelope" title="The user is not subscribed to the newletters" style="color:red"></small> subscribe</button></div>
-           -
-           <small>Expires: {{date_convert(user.expiration)}}</small></h2>
+          -
+          <small *ngIf="user.created_at">Created at : {{date_convert(user.created_at)}} - </small>
```

Comment:
why a dash at the end of line?

Reply:
because there are already a dash between newsletter and and "expires:"

And as I insert created between both, I added one dash.

But we can remove it if you want